src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -280,6 +280,77 @@ Below is a list of available validate commands:
 | `--kubeconfig` `<string>`    | Absolute path(s) to the kubeconfig file(s) (default `$HOME/.kube/config`)
 | `--kubecontext` `<string>`   | Kubeconfig context to use
 
+### `gather`
+
+The `subctl gather` is a data collection tool that gathers Submariner/Kubernetes resources from all clusters in a cluster set by iterating
+over all kubeconfigs. The resources collected are:
+
+* pod logs for
+  * Gateway pods
+  * RouteAgent pods
+  * Globalnet pods
+  * NetworkPluginSyncer pods
+  * ServiceDiscovery pods
+  * CoreDNS pods
+  * Submariner Operator pods
+* ServiceExports
+* ServiceImports
+* EndpointSlices
+* ConfigMaps
+  * submariner-lighthouse-coredns
+  * coredns
+* Endpoints
+* Clusters
+* Submariners
+* ServiceDiscoveries
+* DaemonSet
+  * submariner-gateway
+  * submariner-routeagent
+  * submariner-gateway
+* Deployment
+  * NetworkPluginSyncer Deployment
+  * submariner-lighthouse-agent
+  * submariner-lighthouse-coredns
+  * submariner-operator
+* Below information from `generic`, `canal-flannel`, `weave-net` and `OpenShiftSDN` CNIs. OVN is not supported.
+  * `ip route show`
```

Comment:
May be we don't need to be so specific about what we capture, probably saying that we capture relevant kernel dataplane settings.